src/lib/yang/data.lua

```diff
@@ -16,6 +16,8 @@ function normalize_id(id)
    return id:gsub('[^%w_]', '_')
 end
 
+local path_data = require('lib.yang.path_data')
```

Comment:
I think this is a circular import, isn't it?  A bit unfortunate, though I understand.  I guess one way to fix would be to put ll the code in path_data.lua, then we'll call the checker "automatically" only from the high-level interfaces in lib/yang.lua.  WDYT?  Dunno :)  If not, maybe move this import to inside the consistency_checker function.